package.json

```diff
@@ -3,17 +3,17 @@
   "version": "0.1.0",
   "private": false,
   "dependencies": {
-    "@data-driven-forms/pf4-component-mapper": "^1.23.0",
-    "@data-driven-forms/react-form-renderer": "^1.23.0",
+    "@data-driven-forms/pf4-component-mapper": "^1.23.8",
+    "@data-driven-forms/react-form-renderer": "^1.23.8",
     "@patternfly/react-core": "^3.124.1",
     "@patternfly/react-icons": "^3.14.25",
     "@patternfly/react-table": "^2.24.41",
-    "@redhat-cloud-services/approval-client": "^1.0.44",
+    "@redhat-cloud-services/approval-client": "^1.0.45",
     "@redhat-cloud-services/frontend-components": "0.0.43",
     "@redhat-cloud-services/frontend-components-notifications": "0.0.7",
     "@redhat-cloud-services/frontend-components-utilities": "0.0.15",
-    "@redhat-cloud-services/rbac-client": "^1.0.44",
-    "acorn": "^6.1.1",
+    "@redhat-cloud-services/rbac-client": "^1.0.45",
+    "acorn": "^6.4.0",
```

Comment:
this dependency is not used anywhere.

Reply:
@Hyperkid123  - The build fails here without this, although it succeeds locally.